tests/h/services/search_index/_queue_test.py

```diff
@@ -137,206 +136,194 @@ def add_where(self, queue):
 
 class TestSync:
     def test_it_does_nothing_if_the_queue_is_empty(self, batch_indexer, queue):
-        counts = queue.sync(LIMIT)
+        counts = queue.sync(1)
 
         assert counts == {}
         batch_indexer.index.assert_not_called()
 
     def test_it_ignores_jobs_that_arent_scheduled_yet(
-        self, add_all, batch_indexer, queue
+        self, batch_indexer, now, queue, SyncAnnotationJobFactory
     ):
-        add_all(schedule_in=ONE_WEEK_IN_SECONDS)
+        SyncAnnotationJobFactory(scheduled_at=now + datetime_.timedelta(hours=1))
 
-        counts = queue.sync(LIMIT)
+        counts = queue.sync(1)
```

Comment:
We add one job to the queue (one call to `SyncAnnotationJobFactory()`) and then we call `sync(1)`. There's no need for the `LIMIT` constant. I made this same change (`sync(LIMIT)` to `sync(1)`) in lots of tests. Otherwise it looks like there's something important about `LIMIT`, looks like it's shared between all the tests for a reason. There isn't.